pkg/epp/scheduling/types/types.go

```diff
@@ -30,23 +30,22 @@ type LLMRequest struct {
 	Model string
 	// Target models is a map of target model name to weight.
 	TargetModels map[string]int
+	Prompt       string
```

Comment:
where is it used?

Reply:
Will be used in prefix caching. I can take it out and add it to the prefix caching PR but I'd like to just keep it in this PR.